app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100
```

Comment:
hidden is false by default, but if I want to create an animal with hidden=True I will get an error.